Packs/FeedAutofocus/Integrations/FeedAutofocus/FeedAutofocus.py

```diff
@@ -146,15 +225,19 @@ def module_test_command(client: Client, args: dict):
         try:
             client.build_iterator()
         except Exception:
-            raise Exception("Could not fetch Daily Threat Feed\n\nCheck your API key and your connection AutoFocus.")
+            raise Exception("Could not fetch Daily Threat Feed\n\nCheck your API key and your connection to AutoFocus.")
```

Comment:
maybe we should aggregate the exceptions here? the user might want to add them both..

Reply:
done